src/functions-reference/hidden_markov_models.Rmd

```diff
@@ -1,58 +1,73 @@
 transition matrix is used for all transitions.
 Finally we define the initial state $K$-vector $\rho$, with
 $$
-  \rho_i = p(x_0 = i).
+  \rho_k = p(x_0 = k \mid \phi).
 $$
 
-### Stan functions
+The Stan functions that support this type of model are special
+in that the user does not explicitly pass $y$ and $\phi$ as arguments.
+Instead, the user passes $\log \omega$, $\Gamma$, and $\rho$,
+which in turn depend on $y$ and $\phi$. 
+
+## Stan functions
 
 <!-- real; hmm_marginal; (matrix log_omega, matrix Gamma, vector rho); -->
 \index{{\tt \bfseries hmm\_marginal }!{\tt (matrix log\_omega, matrix Gamma, vector rho): real}|hyperpage}
 
 `real` **`hmm_marginal`**`(matrix log_omega, matrix Gamma, vector rho)`<br>\newline
-The log probability density of y, with the hidden states, $x_n$, integrated out at each iteration, given the matrix of log observational probability densities, $\log \omega$,
-the common transition matrix, $\Gamma$, and the initial state vector,
-$\rho$.
+Returns the log probability density of $y$, with $x_n$ integrated out at each iteration.
+The arguments represent (1) the log density of each output, (2) the transition matrix, and (3) the initial state vector.
+
+*   *`log_omega`*: $\log p(y_n \mid x_n = k, \phi)$, log density of each output,
+
+*   *`Gamma`*: $\Gamma_{ij} = p(x_n = j | x_{n - 1} = i, \phi)$, the transition matrix,
+
+*   *`rho`*: $p(x_0 = k \mid \phi)$, the initial state probability.
+
 
 \index{{\tt \bfseries hmm\_latent\_rng }!{\tt (matrix log\_omega, matrix Gamma, vector rho): int[]}|hyperpage}
 
 `int[]` **`hmm_latent_rng`**`(matrix log_omega, matrix Gamma, vector rho)`<br>\newline
-A sample from the joint posterior distribution of the hidden states,
+Returns an array of integers over $\{1, ..., K\}$,
+sampled from the joint posterior distribution of the hidden states,
 $p(x_{1:N} \mid y_{1:N}, \omega, \Gamma, \rho)$.
 May be only used in transformed data and generated quantities.
 
 \index{{\tt \bfseries hmm\_hidden\_state\_prob }!{\tt (matrix log\_omega, matrix Gamma, vector rho): matrix}|hyperpage}
 
 `matrix` **`hmm_hidden_state_prob`**`(matrix log_omega, matrix Gamma, vector rho)`<br>\newline
-The marginal posterior probabilities of each hidden state value,
-$p(x_n = k \mid \omega, \Gamma, \rho)$ for all iterations 
-$n \in \{1, ..., N \}$, stored in a matrix.
-The $i^\mathrm{th}$ column is a simplex of probabilities for the $i^\mathrm{th}$ variable.
+Returns the matrix of marginal posterior probabilities of each hidden state value. This will be a $K \times N$ matrix.
+<!-- The marginal posterior probabilities of each hidden state value, -->
+<!-- $p(x_n = k \mid \omega, \Gamma, \rho)$ for all iterations  -->
+<!-- $n \in \{1, ..., N \}$, stored in a matrix. -->
+The $i^\mathrm{th}$ column is a simplex of probabilities for the $n^\mathrm{th}$ variable.
 Moreover, let $A$ be the output. Then 
 $A_{ij} = p(x_j = i \mid \omega, \Gamma, \rho)$.
```

Comment:
Add a phi to the end here.

Reply:
The dependence on $\phi$ is mediated by $\omega$, $\Gamma$ and $\rho$. The alternative you be

$p(x_j  = i \mid y, \phi)$

which I'm going to roll with.